dist/challenge-templates/pwn/challenge.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: kctf.dev/v1alpha1
 kind: Challenge
 metadata:
-  name: mychal
+  name: pwn
```

Comment:
should the name be set with `kctf chal create`? I didnt find where this happens

Reply:
found it

https://github.com/google/kctf/blob/d35b42f705db3d481a93831be2c03a4117d6c334/dist/bin/kctf-challenge#L461